Solutions/IllumioSaaS/Playbooks/Illumio-Port-Blocking-Switch/azuredeploy.json

```diff
@@ -0,0 +1,364 @@
+                        },
+                        "PortBlockingFunction-createAllowRuleForVirtualService": {
+                            "runAfter": {
+                                "PortBlockingFunction-bindWorkloadsToVirtualService": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "type": "Function",
+                            "inputs": {
+                                "body": "@body('PortBlockingFunction-bindWorkloadsToVirtualService')",
+                                "function": {
+                                    "id": "[concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Web/sites/',parameters('Functionappname'), '/functions/createAllowRuleForVirtualService')]"
+                                }
+                            }
+                        },
+                        "PortBlockingFunction-changeWorkloadEnforcementState": {
+                            "runAfter": {
+                                "PortBlockingFunction-fetchVisibilityOnlyWorkloadsFromTrafficResults": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "type": "Function",
+                            "inputs": {
+                                "body": "@body('PortBlockingFunction-fetchVisibilityOnlyWorkloadsFromTrafficResults')",
+                                "function": {
+                                    "id": "[concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Web/sites/',parameters('Functionappname'), '/functions/changeWorkloadEnforcementState')]"
+                                }
+                            }
+                        },
+                        "PortBlockingFunction-createDenyRule": {
+                            "runAfter": {
+                                "PortBlockingFunction-fetchVisibilityOnlyWorkloadsFromTrafficResults": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "type": "Function",
+                            "inputs": {
+                                "body": "@body('PortBlockingFunction-fetchVisibilityOnlyWorkloadsFromTrafficResults')",
+                                "function": {
+                                    "id": "[concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Web/sites/',parameters('Functionappname'), '/functions/createDenyRule')]"
+                                }
+                            }
+                        }
+                    },
+                    "outputs": {}
+                },
+                "parameters": {
+                    "$connections": {
```

Comment:
Connections value are empty

Reply:
This doesnt need any connection strings since each workflow item is simply leveraging/ invoking a function from a function app.